### ISSUE TITLE
Auth Profile: Address Change Messaging: Address accessibility, build testing

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#35205]
# Authorized Experience Profile Standard Issue Template

## Background

The modal component looks to have the necessary functionality available to allow us to wire up the address change actions to display more information.

Modal component storybook: https://design.va.gov/storybook/?path=/docs/components-modal--default

Design docs for modal: https://design.va.gov/components/modal

Changing the Home and Mailing address and posting both of those updates at the 'same' time (aka when the modal 'yes' button is clicked) may present some interesting challenges.

We are basically presenting one modal after another. Are their any accessibility/usability concerns to this type of flow?
Wiring up the successive API calls will require some new implementations and testing around. Should a loading indicator appear at some point during these api calls to handle response latency?
Handling the failure of one or both updates may create some confusion, especially is one address update call succeeds and one fails.
At this point the notification that shows up below the address sections once the address is updated is limited to showing one at a time, and we would need the ability to display 2 notifications at the same time (one below mailing and one below home)

## Tasks
- [ ] Address accessibility issues
- [ ] Build testing for all accessibility additions

## Acceptance Criteria
- [ ] Accessibility issues have been addressed
- [ ] Testing for all accessibility additions has been created

## How to configure this issue
- Add label for practice area (frontend, backend, design)
- Assign to a Profile team member

**Comment by adamwhitlock1** [comment]
@SKasimow and @Samara-Strauss I have merged the code for this and it is available in staging for validation. I was able to log in with the vets.gov.user+3@gmail.com test user to validate on staging myself. I had to use the login.gov process instead of id.me since id.me seems to be having issues right now.

![Screen Shot 2022-04-20 at 10.14.29 AM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/890152e9-167b-41be-9ae1-567384895201)

Basically validation would involve updating the home address, then updating the mailing address within the modal dialog, and verifying that the 2 success alerts are present and don't disappear after 5 seconds like they used to.

**Comment by adamwhitlock1** [comment]
Validated this with @SKasimow as working. Closing issue
